src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -55,14 +59,20 @@ private void createStage() {
         spriteBatch = new SpriteBatch();
         stage = new Stage(viewport, spriteBatch);
 
-        final Table infoAndWorldGroup = new Table();
-        infoAndWorldGroup.setFillParent(true);
-        stage.addActor(infoAndWorldGroup);
-
-        infoAndWorldGroup.left();
+        final Table infoAndGameTable = new Table();
+        infoAndGameTable.setFillParent(true);
+        infoAndGameTable.add(infoView.getPane()).fillY().expandY();
+        infoAndGameTable.add(gameContainer);
```

Comment:
Having `gameContainer` be something that contains both the `gameView` (which is actually closer to `worldView` now, right?) and the `toolbarView` makes it sound like the toolbar is a part of the game. 

Since the infoview, gameview and toolbarview are all three elements on the same abstraction level (they all contain sub-views and represent a section of the game window), they should all be put directly into the table. 

Tables are like grids, you should be able to set up two rows and two columns and have the infoview take up two rows, while the others take up one. And then it should be renamed to something more appropriate like `userInterfaceTable` or something.